.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: fontsan CI
+
+on: push
+
+jobs:
+  build_and_test:
+    name: Build and Test
+    strategy:
+      matrix:
+        os:
+          - "ubuntu-latest"
+          - "macos-latest"
+          - "windows-latest"
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+
+      - name: Cargo build
+        run: cargo build -vv
+
+      - name: Cargo test
+        run: cargo test -vv
+        env:
+          RUST_BACKTRACE: 1
+
+  build_result:
+    name: homu build finished
+    runs-on: ubuntu-latest
+    needs:
+      - "build_and_test"
+    steps:
+      - name: Mark the job as successful
+        run: exit 0
+        if: success()
+      - name: Mark the job as unsuccessful
+        run: exit 1
+        if: "!success()"
```

Comment:
These should be modified to take into account all task dependencies. See https://github.com/servo/webrender/blob/master/.github/workflows/main.yml#L53. This is important for the proper functioning of the merge queue.